src/table/body-cell/inline-editor.tsx

```diff
@@ -63,10 +63,7 @@ export function InlineEditor<ItemType>({
       await submitEdit(item, column, currentEditValue);
       setCurrentEditLoading(false);
       finishEdit();
-
-      // Consumers in the past have not always been able to support not specifiying the value so we keep and ignore
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    } catch (e) {
+    } catch {
```

Comment:
Not merge blocking but I would like to hear your thoughts – how do you debug such code?


I have got a few cases where I investigated an issue, I traced it down to such anonymous catch block, but there was no clue how I ended up there. Is there any debug trick to know the currently being thrown error?